lib/driver.ts

```diff
@@ -413,6 +413,7 @@ class AndroidDriver
   activateApp = activateApp;
   queryAppState = queryAppState;
   isAppInstalled = isAppInstalled;
+  mobileBackgroundApp = background;
```

Comment:
for espresso driver the removal of backgroundApp was done deliberately https://github.com/appium/appium-espresso-driver/blob/6634b81d0014774039fd68a87cc141a4f67137ba/CHANGELOG.md?plain=1#L59

The reason for it is that espresso server, being integrated into the app under test, stops working if the app is not in foreground, so it makes no sense to have this API available there